diff --git a/node_modules/react-native/Libraries/ReactNative/PaperUIManager.js b/node_modules/react-native/Libraries/ReactNative/PaperUIManager.js
index efb474b..943243b 100644
--- a/node_modules/react-native/Libraries/ReactNative/PaperUIManager.js
+++ b/node_modules/react-native/Libraries/ReactNative/PaperUIManager.js
@@ -17,7 +17,7 @@ import nullthrows from 'nullthrows';
 const NativeModules = require('../BatchedBridge/NativeModules').default;
 const defineLazyObjectProperty =
   require('../Utilities/defineLazyObjectProperty').default;
-const Platform = require('../Utilities/Platform').default;
+const Platform = require('../Utilities/Platform');
 const UIManagerProperties = require('./UIManagerProperties').default;
 
 const viewManagerConfigs: {[string]: any | null} = {};
