diff --git a/node_modules/react-native/Libraries/ReactNative/BridgelessUIManager.js b/node_modules/react-native/Libraries/ReactNative/BridgelessUIManager.js
--- a/node_modules/react-native/Libraries/ReactNative/BridgelessUIManager.js
+++ b/node_modules/react-native/Libraries/ReactNative/BridgelessUIManager.js
@@ -18,7 +18,7 @@
-import Platform from '../Utilities/Platform';
+import { Platform } from 'react-native';
